doc/api/process.markdown

```diff
@@ -116,6 +116,45 @@ Nine out of ten times nothing happens - but the 10th time, your system is bust.
 
 You have been warned.
 
+## Event: 'unhandledRejection'
+
+Emitted whenever a `Promise` is rejected and no error handler is attached to the promise within a turn of the event loop. When programming with promises exceptions are encapsulated as rejected promises. Such promises can be caught and handled using `promise.catch(...)` and rejections are propagated through a promise chain. This event is useful for detecting and keeping track of promises that were rejected whose rejections were not handled yet. This event is emitted with the following arguments:
+
+ - `reason` the rejection reason of the promise susprected in having an unhandled rejection
+ - `p` the promise suspected in having an unhandled rejection itself.
+
+Here is an example that logs every unhandled rejection to the console
+
+    process.on('unhandledRejection', function(reason, p){
+        console.log("Possibly Unhandled Rejection at: Promise ", p, " reason: ", reason);
+        // application specific logging, throwing an error, or other logic here
+    });
+
+For example, here is a rejection that will be trigger the `"unhandledRejection"` event:
+
+    somePromise.then(function(res){
+      return reportToUser(JSON.pasre(res)); // note the typo
+    }); // no `.catch` or `.then`
+    
+## Event: 'rejectionHandled'
+
+Emitted whenever a Promise was rejected and an error handler was attached to it later than after an event loop turn. This event is emitted with the following arguments:
+
+ - `p` the promise who was believed to be an unhandled rejection. 
```

Comment:
the promise that was previously emitted in an `"unhandledRejection"` event, but who has now gained a rejection handler